test/fake-repos/fake-proto/excluded/excluded.proto

```diff
@@ -0,0 +1,8 @@
+// This is a fake proto file for testing.
```

Comment:
I have added test cases that check for the returned value of find_proto methods so the content of the proto should not matter. Consider done.